build.gradle

```diff
@@ -66,7 +66,7 @@ spotless {
     }
     sql {
         target '**/*.sql'
-        targetExclude "**/build/**/*", "**/.gradle/**/*", "**/.venv/**"
+        targetExclude "**/build/**/*", "**/.gradle/**/*", "**/.venv/**", "airbyte-integrations/bases/base-normalization/dbt-project-template/**/*"
```

Comment:
If you merge master in your branch, I will be able to update this code to be working with the faster spotless